### PR TITLE
Start of AnomalyKit server-side code import.

### PR DESCRIPTION
**Pull request by cmccoy**
Adds AnomalyKit code to `/alerts`.

All tests now pass.

To resolve before merging:

- [x] Decide how to structure the code (#6)
- [x] Fix code duplicated in `alerts/perfkit/common`.
- [x] Fix failing unit tests.

### CODE REVIEW

**Comment by cmccoy** [comment]
Getting closer:

* The `perfkit.common` package is now under `/common`.
* `/server` was renamed to `/explorer`.
* A gulp task adds the `perfkit.common` package to `alerts` and `explorer`.

**Comment by cmccoy** [comment]
@jmuharsky: unit tests pass; this seems like a reasonable point to merge in and start making smaller changes to integrate.

To run alerts unit tests: `./compile.sh && ./deploy/alerts/test/build-and-unit-test.sh`

**Comment by jmuharsky** [comment]
Need to add rimraf to your package.json.

File: README.md [hunk 1]
```diff
@@ -140,7 +136,12 @@ Compile and Deploy PerfKit Explorer
 
 1. Deploy PerfKit Explorer to App Engine.
 
-         appcfg.py --oauth2 update deploy
+         gcloud preview app deploy --project MY_PROJECT_ID --version MY_VERSION \
```
Comment (jmuharsky) [inline README.md]: deploying per the instructions yields a 400 error with a known project.

File: alerts/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Setting up a development environment.
```
Comment (jmuharsky) [inline alerts/README.md]: These instructions should be covered in the top-level README.MD

File: alerts/alerts.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    target_utilization: 0.5
+
+builtins:
+- remote_api: on
+- appstats: on
+
+# Dispatch table from URL -> application instance or from URL -> static content.
+handlers:
+# Angular Templates
+#- url: /alerts/_static/(.*\.html)
```
Comment (jmuharsky) [inline alerts/alerts.yaml]: Remove unused handlers.

**Comment by cmccoy** [comment]
> Need to add rimraf to your package.json.
It's already a development dependency: https://github.com/cmccoy/PerfKitExplorer-private/blob/alert-backend/package.json#L17
Is there something else I need to do?

**Comment by jmuharsky** [comment]
I suppose not.  I see it in package.json, but I had run npm package update as well, and didn't get it.

**Comment by jmuharsky** [comment]
LGTM

**Comment by cmccoy** [comment]
Since the deployment issues seem like an MVM bug, I'm merging this (but just into the `alerts` branch).

**Pull request merged**